components/pageDescription/PageDescription.styles.js

```diff
@@ -0,0 +1,68 @@
+import { P } from '../../style/typography';
+import styled from 'styled-components';
+import { min } from '../../lib/responsive';
+
+const PageDescriptionContainer = styled.div`
+  display: flex;
+  position: relative;
+  @media ${min.mobile} {
+    padding: 4em 0 0 0;
+    margin: 0 0 0 10em;
+  }
+  @media ${min.tablet} {
+    padding: 2em 0 0 0;
+    margin: 0 0 0 8em;
+  }
+  @media ${min.desktop} {
+    padding: 3em 0 0 0;
+    margin: 0 0 0 11em;
+  }
+`;
+
+const DescriptionContainer = styled.div`
+  margin: 0 0 0 2em;
+`;
+
+const Description = styled(P)`
+  @media ${min.mobile} {
+    margin: 0 2em 0 2em;
+    width: 18em;
+    line-height: 1.8em;
+  }
+  @media ${min.tablet} {
+    margin: 0 0 0 2em;
+    width: 15em;
+    line-height: 1.8em;
```

Comment:
similar to my comment [here](https://github.com/Scout-NU/thealleyway-site/pull/84/files#r612099791), line 35 & 40 will still be applied from 30